### DISCUSSION TITLE
Question: How can I quit the censure???

### DISCUSSION THREAD

**Discussion by LaxeDragon426** [FunkinCrew/Funkin#1558] [Q&A]

**Comment by ghost** [comment]
What do you mean?

**Comment by LaxeDragon426** [comment]
i said, characters like BF, GF, Mom an Monster is only a black square, this happen with the background of Week 2

**Comment by LaxeDragon426** [comment]
![Captura de pantalla 2021-05-24 144756](https://user-images.githubusercontent.com/84741144/119399889-126eb800-bc9f-11eb-88e1-d9e9d0e05dc9.png)

**Comment by ghost** [comment]
Looks like you don't have enough RAM

**Comment by LaxeDragon426** [comment]
and how i fix it??

**Comment by ghost** [comment]
There's not much you can do without upgrading your computer

**Comment by LaxeDragon426** [comment]
oh... i see
